transformer_engine/pytorch/attention.py

```diff
@@ -529,36 +555,25 @@ def forward(
         """
 
         use_flash_attention = self.use_flash_attention
-        if (query_layer.dtype not in [torch.bfloat16, torch.float16]
+        if (query_layer.dtype not in [torch.bfloat16, torch.float16] # pylint: disable=too-many-boolean-expressions
```

Comment:
Could probably use `any(x.dtype not in [torch.bfloat16, torch.float16] for x in [query_layer, key_layer, value_layer])`. I think pylint recognizes this as one boolean expression.